### PR TITLE
Fix GetTables operation in Flink

### PR DESCRIPTION
**Pull request by cxzl25**
### _Why are the changes needed?_
#1646

In Flink, the schema returned by `GetTables` is incorrect, causing some database management tools to not see the table.

### _How was this patch tested?_
- [x] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [x] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

**Comment by cxzl25** [comment]
## Fix
image

File: externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/GetTables.scala [hunk 1]
```diff
@@ -21,50 +21,79 @@ import scala.collection.JavaConverters._
...
+                  flinkTable.getComment,
+                  null,
+                  null,
+                  null,
+                  null,
+                  null)
+              }
           }
-      }
+      }.toArray
```
Comment (pan3793) [inline externals/kyuubi-flink-sql-engine/src/main/scala/org/apache/kyuubi/engine/flink/operation/GetTables.scala:81]: nit: change line 51 `toSeq` to `toArray` could avoid this transformation

**Review by pan3793 — APPROVED** [review]
Thanks for the investigation and fixing, LGTM, only minor comments

**Comment by pan3793** [comment]
Thanks, merging to master/1.6

**Pull request closed without merging**